nilearn/plotting/surface/tests/test_surf_plotting.py

```diff
@@ -877,6 +894,55 @@ def test_plot_surf_parcellation_plotly(
     )
 
 
+@pytest.mark.parametrize("avg_method", ["mean", "median"])
+@pytest.mark.parametrize("symmetric_cmap", [True, False, None])
+def test_plot_surf_roi_default_arguments(
+    plt, engine, symmetric_cmap, avg_method, surface_image_roi
+):
+    """Regression test for https://github.com/nilearn/nilearn/issues/3941."""
+    # To avoid extra warnings
+    if engine == "plotly":
+        avg_method = None
+
+    plot_surf_roi(
+        surface_image_roi.mesh,
+        roi_map=surface_image_roi,
+        engine=engine,
+        symmetric_cmap=symmetric_cmap,
+        darkness=None,  # to avoid deprecation warning
+        cmap="RdYlBu_r",
+        avg_method=avg_method,
+    )
+
+
+@pytest.mark.parametrize(
+    "kwargs", [{"vmin": 2}, {"vmin": 2, "threshold": 5}, {"threshold": 5}]
+)
+def test_plot_surf_roi_colorbar_vmin_equal_across_engines(
+    matplotlib_pyplot, plotly, kwargs, in_memory_mesh
+):
+    """Regression test for https://github.com/nilearn/nilearn/issues/3944."""
+    roi_map = np.arange(0, len(in_memory_mesh.coordinates))
+
+    mpl_plot = plot_surf_roi(
+        in_memory_mesh,
+        roi_map=roi_map,
+        colorbar=True,
+        engine="matplotlib",
+        **kwargs,
+    )
+    plotly_plot = plot_surf_roi(
+        in_memory_mesh,
+        roi_map=roi_map,
+        colorbar=True,
+        engine="plotly",
+        **kwargs,
+    )
+    assert (
+        mpl_plot.axes[-1].get_ylim()[0] == plotly_plot.figure.data[1]["cmin"]
+    )
+
+
```

Comment:
I've just moved these 2 tests up in the file to keep all `plot_surf_roi` function tests together.